src/legacy/status_im/keycard/card.cljs

```diff
@@ -401,15 +401,12 @@
    (merge
     args
     {:on-success
-     (fn [response]
+     (fn [_response]
        (log/debug "[keycard response succ] export-key")
-       (re-frame/dispatch [:keycard.callback/on-export-key-success
-                           response]))
+       (js/alert "feature no longer supported"))
```

Comment:
discussed this @cammellos - 
as I understand Keycard is not hooked up at all and we soon will need to reimplement it with the new ui. 
For that reason these changes are "safe" - I will add a follow up pr to remove the keycard related pieces completely.

Reply:
Maybe alert "feature under development" instead of "feature no longer supported" which can imply we have no intention of supporting Keycard in the future

Reply:
yeah good point, I think the screen is no longer viewable but to be on the safe side I will change it to that.